client/src/components/home/home.scss

```diff
@@ -0,0 +1,139 @@
+.home-bg {
+  font: 100% $font-stack;
+  color: $primary-color;
+  background-image: url('https://res.cloudinary.com/djvjxp2am/image/upload/v1510227221/newbg_c6e8fv.png');
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  min-height: 100vh;
+  display: flex;
+  -webkit-display: flex;
+  flex-direction: row;
+  -webkit-flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+
+.form-holder {
+  min-width: 360px;
+  height: auto;
+  margin: auto;
+  margin-bottom: 35%;
+
+  .btn {
+    width: 100%;
+  }
+
+  .form-header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    -webkit-flex-direction: row;
+    color: white;
+
+    h2 {
+      padding: 0;
+      margin: 0, 0,5px;
+    }
+
+    div {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+
+        a, p {
+          color: white;
+          margin-bottom: -5%; 
+        }
+        a:hover {
+          color: white;
+          text-decoration: underline;
+        }
+       
+    }
+
+  }
+
+  .user-form {
+    // background: #F9F9F9;
```

Comment:
If you're not using the styles then no need to keep them

Reply:
Alright Femi